apps/site/assets/ts/schedule/components/line-diagram/StopCard.tsx

```diff
@@ -122,16 +127,19 @@ const StopCard = (props: StopCardProps): ReactElement<HTMLElement> => {
             )
           )}
         </div>
-
-        <footer className="m-schedule-diagram__footer">
-          <button
-            className="btn btn-link"
-            type="button"
-            onClick={() => onClick(routeStop)}
-          >
-            View schedule
-          </button>
-        </footer>
+        {(!liveData?.headsigns.length &&
```

Comment:
suggestion (blocking): Between showing/hiding the footer and the button label variations, I think this file could use a few test cases, please.